src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -351,14 +351,32 @@ public function findMany($ids, $columns = ['*'])
      * Find a model by column, operator and value.
      *
      * @param  string  $column
-     * @param  string  $operator
-     * @param  mixed  $value
+     * @param  string|null  $operatorOrValue
+     * @param  mixed|null  $valueOrColumns
      * @param  array  $columns
      * @return \Illuminate\Database\Eloquent\Model|null
      */
-    public function findBy($column, $operator, $value, $columns = ['*'])
+    public function findBy($column, $operatorOrValue = null, $valueOrColumns = null, array $columns = ['*'])
     {
-        return $this->where($column, $operator, $value)->first($columns);
+        $args = func_get_args();
+
+        if (count($args) < 2) {
+            return;
+        }
+
+        if (count($args) == 2) {
```

Comment:
I think, you should change `==` to `===` and same with line 371

Reply:
It can be changed, but I don't think it matters in this case